test/ad/distributions.jl

```diff
@@ -409,12 +409,11 @@
             # Matrix case does not work with Skellam:
             # https://github.com/TuringLang/DistributionsAD.jl/pull/172#issuecomment-853721493
             filldist_broken = if D <: Skellam
-                ((d.broken..., :Zygote, :ReverseDiff), (d.broken..., :Zygote, :ReverseDiff))
+                ((d.broken..., :ReverseDiff), (d.broken..., :ReverseDiff))
             elseif D <: PoissonBinomial
                 ((d.broken..., :Zygote), (d.broken..., :Zygote))
             elseif D <: Chernoff
-                # Zygote is not broken with `filldist`
-                ((), ())
+                ((d.broken..., :Zygote), (d.broken..., :Zygote))
```

Comment:
`Chernoff` is already marked as broken for `:Zygote`. The branch explicitly removed this since `filldist` was not broken with Zygote. But if this PR breaks `filldist` with `Zygote` you can just remove the whole branch.
```suggestion
```